crypto-ffi/src/generic/mod.rs

```diff
@@ -1650,3 +1651,21 @@ mod tests {
         });
     }
 }
+
+pub struct DatabaseKey(core_crypto_keystore::DatabaseKey);
+
+impl DatabaseKey {
+    // TODO: This is here only because we need to construct an instance in interop.
+    // Remove it once we drop the FFI client from interop.
+    pub fn new(key: core_crypto_keystore::DatabaseKey) -> DatabaseKey {
```

Comment:
could be `#[derive(derive_more::Constructor)]`

Reply:
Nice, didn't know that, thanks!